### PR TITLE
Issue 2461: moved supported browsers to the rel notes

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://github.com/openshift/openshift-docs/issues/2461

Should be straight-forward, but @adellape @ahardin-rh @tpoitras please let me know if there's any issues.

### CODE REVIEW

**Comment by bfallonf** [comment]
Actually, @adellape have you been working on the 3.3 rel notes? Can we make sure the move here sticks around for 3.3?

**Comment by bfallonf** [comment]
@adellape bump!

File: release_notes/ose_3_2_release_notes.adoc [hunk 1]
```diff
@@ -619,6 +619,35 @@ behavior, run:
 $ oc rsh <name> -- /bin/bash
 ----
 
+[[browser-requirements]]
```
Comment (adellape) [inline release_notes/ose_3_2_release_notes.adoc]: Maybe give this a more specific ID like `ose-3-2-browser-requirements`?

File: release_notes/ose_3_2_release_notes.adoc [hunk 2]
```diff
@@ -619,6 +619,35 @@ behavior, run:
...
+== Browser Requirements
+
+ifdef::openshift-enterprise[]
+Review the https://access.redhat.com/articles/2176281[tested integrations] for
+{product-title}.
+endif::openshift-enterprise[]
```
Comment (adellape) [inline release_notes/ose_3_2_release_notes.adoc]: Can drop `openshift-enterprise` from the `endif`.

**Review by adellape — COMMENTED** [review]
Whoa, an oldie! Couple minor things but LGTM in general. Maybe we should look to do this for 3.3 and later after, as well.

File: release_notes/ose_3_2_release_notes.adoc (hunk 2 above)
Comment (adellape) [inline release_notes/ose_3_2_release_notes.adoc]: Well actually, you could drop the `ifdef` completely, since this is an enterprise-only topic overall.

**Comment by ahardin-rh** [comment]
@bfallonf Looks like this will also address https://github.com/openshift/openshift-docs/issues/4218 if we can get the list updated for each version of OCP.

**Comment by ahardin-rh** [comment]
@bfallonf bump

**Comment by bfallonf** [comment]
@ahardin-rh @adellape Oop this is an old one. Hmm if these are the minimum requirements, they probably haven't changed? Would it be accurate to say this should be in the master branch, then cherry-pick it to each release branch (after making Alex's suggested changes above)?

**Comment by adellape** [comment]
@bfallonf @ahardin-rh Browser support actually came up again recently: jwforres was telling erich and I that the [OpenShift Container Platform Tested Integrations](https://access.redhat.com/articles/2176281) can be updated to include the mobile browsers supported, as well (have that in email). We already link to that KBase as far back as 3.0 docs, so I think we should just remove the table here and let the KBase handle keeping that support matrix updated per OCP version.

The only other issue may be that the KBase only specifically mentions the OCP distro by name (and version #), and not Origin/Online/Dedicated... I guess we'd need to either A) get the KBase updated to address those distros as well or create separate KBases for each (work with GSS), B) just link to it anyway, or C) update and maintain tables here for those distros.

**Comment by bfallonf** [comment]
@adellape sure. Hmm not sure which option is better. I agree that it would be easier to just link to the KBase. No point in doubling-up the efforts.

I'm not too sure what to do next. Is there someone in GSS that would be best to include the distro info in the KBase?

**Comment by adellape** [comment]
> Is there someone in GSS that would be best to include the distro info in the KBase?

@bfallonf We'd wanna chat w/ erich about it.

**Comment by ahardin-rh** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/5745 after discussion with the web console team

**Pull request closed without merging**